assets/styles/styles.scss

```diff
@@ -16,6 +16,6 @@ $black: #000000;
 div {
   background-color: $blue;
   color: $black;
-  border: 2px solid #FFFFFF 
+  border: 2px solid #FFFFFF;   
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`<br>Line contains trailing whitespace